tools/measurement/latency/src/test_target_id_map.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Map } from 'immutable';
+
+export const TEST_TARGET_ID_MAP: Map<string, number> = Map({
+  'postsubmit-gob-sync': 1,
```

Comment:
Some of these run on post-submit, and others, are non-blocking pre-submit. It may be useful to gather these, but we need to exclude them from total pre-submit time.

Reply:
I think eventually we will need to collect them, e.g. for greenness calculation. Do you know how I can tell whether a presubmit job is a blocking one or a non-blocking one?